.github/workflows/new_lambdas-deploy-to-perf.yml

```diff
@@ -17,28 +17,24 @@ permissions:
   id-token: write # This is required for requesting the JWT
   contents: read # This is required for actions/checkout
 
-env:
-  PYTHON_VERSION: "3.11"
-  ENVIRONMENT: perf
-  BUILD_BRANCH: ${{ inputs.build_branch }}
-  SANDBOX: ${{ inputs.sandbox }}
-
 jobs:
   run_tests:
     uses: ./.github/workflows/new_base-lambdas-reusable-test.yml
     with:
-      environment: ${{ env.ENVIRONMENT }}
-      python-version: ${{ env.PYTHON_VERSION }}
-      build-branch: ${{ env.BUILD_BRANCH }}
+      environment: perf
```

Comment:
Is there a reason why we're hard coding this and subsequent environments instead of using `inputs.environment`

Reply:
Cheers for taking a quick look, but yup this one is specific to perf so we don't want to open this one up to additional environments that can be input at runtime.